### ISSUE TITLE
LDAP is logging error message

### ISSUE THREAD

**Issue by gig13** [owncloud/core#18563]
Reference https://github.com/owncloud/core/issues/17627 for all config information.

The log repeatedly shows:
```
{PHP} Non-static method
OCA\user_ldap\lib\Helper::getServerConfigurationPrefixes() should not be called statically at
/var/www/html/owncloud/apps/files_ldap_home/lib/storage.php#81
```

Changing this function (public function
getServerConfigurationPrefixes) to (public static function getServerConfigurationPrefixes) in apps/user_ldap/lib/helper.php appears to fix the problem.  This may have other consequences, but everything seems to work.

**Comment by gig13** [comment]
@MorrisJobke

**Comment by MorrisJobke** [comment]
Fuplicate of #779

Fixed in 8.1.2 and 8.0.7 (ETA next tuesday)
